<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgFinishLocate</class>
 <widget class="QDialog" name="CDlgFinishLocate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mouth Locate Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Methods:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboMethod"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnOutRing">
       <property name="text">
        <string>OutterCircle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnInRing">
       <property name="text">
        <string>InnerCircle</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupRing">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="labelEdge">
        <property name="text">
         <string>Edge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinEdge">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelOpenSize">
        <property name="text">
         <string>OpenSize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spinOpenSize">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupOther">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="labelLowThres">
        <property name="text">
         <string>LowThres</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinLowThres">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelHighThres">
        <property name="text">
         <string>HighThres</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinHighThres">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelOffset">
        <property name="text">
         <string>CenOffset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinCenOffset">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBack">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="labelFloatRange">
        <property name="text">
         <string>FloatingRange</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinFloatRange">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPercent">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>118</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboMethod</tabstop>
  <tabstop>btnOutRing</tabstop>
  <tabstop>btnInRing</tabstop>
  <tabstop>spinEdge</tabstop>
  <tabstop>spinOpenSize</tabstop>
  <tabstop>spinFloatRange</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
